plugin/trino-blackhole/src/main/java/io/trino/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -408,4 +413,46 @@ public void setViewColumnComment(ConnectorSession session, SchemaTableName viewN
                 view.getOwner(),
                 view.isRunAsInvoker()));
     }
+
+    @Override
+    public boolean delegateMaterializedViewRefreshToConnector(ConnectorSession session, SchemaTableName viewName)
+    {
+        return false;
+    }
+
+    @Override
+    public void dropMaterializedView(ConnectorSession session, SchemaTableName viewName)
+    {
+        materializedViews.remove(viewName);
+    }
+
+    @Override
+    public void createMaterializedView(ConnectorSession session, SchemaTableName viewName, ConnectorMaterializedViewDefinition definition, boolean replace, boolean ignoreExisting)
+    {
+        materializedViews.put(viewName, definition);
```

Comment:
i see the funcionality is being tested with iceberg in subsequent commit, so all we need to do is just remove the blackhole implementation.

Reply:
BlackHole implementation was added due to `TestAccessControl`, not sure it will be just enough to remove it.

Reply:
the implementation isn't correct, so we need to remove it

Reply:
Sure, so what is the good way of adjusting this ?
- remove completely support MV for BlackHole
- disable  MV column comment test in `TestAccessControl`

Reply:
> remove completely support MV for BlackHole

that;'s what we have to do 

> disable MV column comment test in TestAccessControl

this i don't know. maybe there is a way to change the test. 
for example, it could use iceberg connector 🤷‍♂️